src/core/mount.c

```diff
@@ -1765,6 +1766,27 @@ static int mount_dispatch_io(sd_event_source *source, int fd, uint32_t revents,
         Iterator i;
         Unit *u;
         int r;
+        uint64_t this_run = now(CLOCK_MONOTONIC);
+
+        if (source && this_run < m->mount_last_read + m->mount_last_duration * 10) {
```

Comment:
this is going to explode if mount_last_duration has not been initialized yet, no?